<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SugarCRM Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCRMAccounts"/>
    <addaction name="separator"/>
    <addaction name="actionImportContacts"/>
    <addaction name="separator"/>
    <addaction name="actionSynchronize"/>
    <addaction name="actionOfflineMode"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfigureFatCRM"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutFatCRM"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCRMAccounts">
   <property name="text">
    <string>CRM Accounts...</string>
   </property>
   <property name="toolTip">
    <string>Manage CRM Accounts</string>
   </property>
  </action>
  <action name="actionSynchronize">
   <property name="text">
    <string>Synchronize</string>
   </property>
   <property name="toolTip">
    <string>Synchronize current CRM Account</string>
   </property>
   <property name="statusTip">
    <string>Synchronize current CRM Account</string>
   </property>
  </action>
  <action name="actionOfflineMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Offline Mode</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionImportContacts">
   <property name="text">
    <string>Import Contacts...</string>
   </property>
  </action>
  <action name="actionConfigureFatCRM">
   <property name="text">
    <string>Configure FatCRM...</string>
   </property>
  </action>
  <action name="actionAboutFatCRM">
   <property name="text">
    <string>About &amp;FatCRM...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
